### ISSUE TITLE
Data normalization point and debugging

### ISSUE THREAD

**Issue by securityedge** [wazuh/wazuh#4317]
|Wazuh version|Component|Install type|Install method|Platform|
|3.9.4|wazuh decoders?|Manager|Packages|Debian 10|

I`m deploy testing wazuh + elk lab. In test lab windows data i`m view in following form:
![image](https://user-images.githubusercontent.com/57942327/69977046-118e1780-153b-11ea-8e65-fee83fdd78cd.png)

In prod
![image](https://user-images.githubusercontent.com/57942327/69977063-18b52580-153b-11ea-8a2a-7a37583b5393.png)
Questions:
Where i`m make misconfig and data not transform?

**Comment by securityedge** [comment]
I learned to influence the fields, i.e. all these fields are uniquely defined in ruleset / decoders / 0380-windows_decoders.xml
![image](https://user-images.githubusercontent.com/57942327/69982020-89ad0b00-1544-11ea-9744-778c7bbaefc8.png)

How to understand why this rule does not apply?

**Comment by securityedge** [comment]
I`m make full diff configs and rules from lab and production and partially resolve issue. Lab in using wazuh 10.2 due to my mistake. 

DevTeam its possible to  make decoding like in screenshot#1 in wazuh 3.9.4-1?

**Comment by Zenidd** [comment]
Hello @securityedge,

Seems like your test environment is getting the events through `Eventlog` monitoring, while your production environment is getting the events through `Eventchannel`.  [Here](https://documentation.wazuh.com/3.10/user-manual/capabilities/log-data-collection/how-to-collect-wlogs.html#windows-eventlog-vs-windows-eventchannel) you can read more about the `Eventlog` and `Eventchannel` capabilities and its comparison. 

For a better diagnose, could you paste here the lab and prod environment ossec.conf? I think that your lab environment has the further Eventlog localfile block:
```
<localfile>
    <location>Security</location>
    <log_format>eventlog</log_format>
</localfile>
```
On the other hand, a modification on the decoders like the one in your message above will only apply to Eventlog events as the Eventchannel fields are dynamically decoded, meaning that there are not customizable decoders and just every field is extracted for each event. 

To summarize:
- Could you paste here your lab and prod Wazuh manager `ossec.conf` file?
- Do you want to use `Eventchannel or Eventlog`? (We recommend using Eventchannel, you can read our related documentation [here](https://documentation.wazuh.com/3.10/user-manual/capabilities/log-data-collection/how-to-collect-wlogs.html#windows-eventlog-vs-windows-eventchannel))

With this information, I'll be able to provide the right guidance for your use case. Greetings, 
JP Sáez

**Comment by securityedge** [comment]
issue is resolved. thx.
